server/app.ts

```diff
@@ -18,7 +19,7 @@ import setUpWebSession from './middleware/setUpWebSession'
 
 import routes from './routes'
 import type { Services } from './services'
-import populateCurrentPrisoner from './middleware/populateCurrentPrisoner'
+//  import populateCurrentPrisoner from './middleware/populateCurrentPrisoner'
```

Comment:
dont we need this any more??

Reply:
No, it gets stuck in the journey session so the API is now only called once per route entry rather than every page load